demo/scss/demo.scss

```diff
@@ -1,5 +1,11 @@
 @import 'carbon-components/scss/globals/scss/styles';
 
+@import '../../src/components/card/card';
```

Comment:
Should we introduce a new .scss that imports all .scss of this addon repo? (Similar to what `carbon-components/scss/globals/scss/styles` does)

Reply:
Great call, adding in now 😄